rudder-webapp/SOURCES/rudder-node-to-relay

```diff
@@ -36,4 +36,10 @@
 
 echo "INFO: This command is deprecated, please use 'rudder server node-to-relay <UUID>'"
 echo -e "Running: rudder server node-to-relay $@ \n"
-rudder server node-to-relay $@
+
+if ! [ "$(command -v rudder server node-to-relay >/dev/null)" ]; then
```

Comment:
What is this supposed to do? To detect the plugin presence it's probably better to query `rudder package` installed plugins list.

Reply:
since we redirect to the command, it's likely what we want to test here. Not sure if it's the correct way to do it, but it seems like it's the correct thing to test.